src/shortcuts.rs

```diff
@@ -74,12 +87,13 @@ macro_rules! with_dollar_sign {
 ///     }
 /// }
 ///
-/// fn view(model: &Model) -> Node<Msg> {
+/// fn view(mdl: &Model) -> Node<Msg> {
 ///     a![
-///         attrs!{ At::Href => Urls::new(base_url).home() }
+///         attrs!{ At::Href => Urls::new(mdl.base_url.clone()).home() }
```

Comment:
Perhaps `&mdl.base` would work here instead of cloning (haven't tried it, mark as resolved if not possible/not important)